legal/formatters.js

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line
+const BigNumber = require("bignumber.js");
```

Comment:
is this duplicated in `commitment` page repo?

Reply:
this particular no — commitment is in typescript. These functions are also in commitment page but generally, there are more of them.